charts/launch-agent/templates/secret.yaml

```diff
@@ -1,16 +1,20 @@
 {{- if not .Values.agent.useExternalWandbSecret }}
+{{- $root := . -}}
+{{- $unique := uniq (append .Values.additionalTargetNamespaces .Values.namespace) }}
+{{- range $ns := $unique }}
 ---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: wandb-api-key-{{ .Release.Name }}
-  namespace: {{ .Values.namespace }}
+  name: wandb-api-key-{{ $root.Release.Name }}
+  namespace: {{ $ns }}
```

Comment:
let's add a label here: `wandb.ai/created-by: helm-chart` so we can distinguish it from an agent-installed secret

similarly, when the agent creates the secret let's set a different label: `wandb.ai/created-by: launch-agent`

this gives us the ability to positively identify these resources were created by us

Reply:
+1, great idea